Khan/wonder-blocks
# [patch:modal] WB-566 Modals and Portals example

## Changes
- Adds an example verifying the parent of a modal be able to update the contents within the modal.
- Small styling tweaks in documentation.

## Test plan
- Open:
    - Staging: https://deploy-preview-403--wonder-blocks.netlify.com/#!/Modal/3
    - Local Dev: http://localhost:6060/#!/Modal/3
- Click on `Open Modal` and close it multiple times.
- Verify the modal title and content are updated every time the modal is reopened.
